articles/architecture-scenarios/b2c/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Introduction to Consumer Identity and Access Management Implementation
+description: High-level overview of the Consumer Identity and Access Management Implementation guide
+toc: false
+topics:
+    - b2c
+    - ciam
+contentType: concept
+useCase:
+  - implementation
+---
+# The Consumer Identity and Access Management Implementation Guide: An Introduction
+
+This guide focuses on showing you how to integrate Auth0 within a Consumer Identity and Access Management (CIAM)-style project. Our recommendations are based on real-world customer implementation experience, and we hope that the advice we provide helps you set your project up for success.
+
+Customers implementing Auth0 for consumer-focused (or Business-to-Consumer (B2C)) projects typically share a common set of goals and objectives. In the sections that follow, we focus on our experiences working with this shared set of expectations. We hope our experience can help you deliver your solution effectively.
```

Comment:
```suggestion
Customers using Auth0 for consumer-focused -  a.k.a. Business-to-Consumer (B2C) - projects typically share a common set of goals and objectives, and in the sections that follow we'll focus on our experiences working with this shared set of expectation to help you deliver your solution effectively and efficiently.
```

Reply:
I'm also wondering if this needs to be a separate paragraph or if we can't just append to the one above?

Reply:
We will leave it separate for scanability/readability on smaller devices.